drivers/soundwire/generic_bandwidth_allocation.c

```diff
@@ -287,7 +343,7 @@ static int sdw_compute_port_params(struct sdw_bus *bus)
 
 	/* Compute transport parameters for grouped streams */
 	ret = sdw_compute_group_params(bus, params,
-				       &group.rates[0], group.count);
+				       &group.rates[0], &group.lanes[0], group.count);
```

Comment:
nitpick: could we just pass ```group``` to this API ?

Reply:
> nitpick: could we just pass `group` to this API ?

Yes, updated